src/components/Faq/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {FaqStyleTitle, FaqStyleContent} from './styles'
+
+class FAQ extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      clickedOn: false
+    }
+    this.handleClick = this.handleClick.bind(this);
+  }
+  
+  handleClick() {
+    // var content = this.nextElementSibling;
+    // console.log("CONTENT STYLE", content.style.maxHeight);
+    this.setState({
+      clickedOn: !this.state.clickedOn
+    });
+  }
+  render() {
+    const {title, answer} = this.props;
+    const questionSelected = this.state.clickedOn;
+    let showAnswer;
+    if (questionSelected) {
+      showAnswer = <div> {answer.answer}</div>
+    }
```

Comment:
I would actually recommend doing a conditional render in the return statement itself. This showAnswer variable is doing double duty if your clickedOn state. So instead of having this live outside the return, you could instead do it inline with some awesome curly braces: 

`return (
      <div>
        <FaqStyleTitle onClick={this.handleClick}> 
          {title}
        </FaqStyleTitle>
        <FaqStyleContent> 
          {
              clickedOn && <div>{answer.answer}</div>
           }
        </FaqStyleContent>
      </div>
      )`